python/tests/db/test_distributed_read.py

```diff
@@ -0,0 +1,105 @@
+import copy
+import numpy as np
+import pymongo
+import sys
+
+from bson.objectid import ObjectId
+from datetime import datetime
+
+sys.path.append("python/tests")
+from mspasspy.db.database import Database, read_distributed_data
+from helper import (get_live_seismogram,
+                    get_live_timeseries)
+
+
+
+def test_read_distributed_data(spark_context):
+    Database.__bases__ = (pymongo.database.Database,)
+    client = pymongo.MongoClient('localhost')
```

Comment:
The above two lines are not necessary with pymongo. Also, instead of creating MongoClient, we should just use our mspasspy.db.client.Client here.

Reply:
done.